tools/check-links

```diff
@@ -87,9 +87,9 @@ class Crawler:
     def get_urls_from_page(self, soup) -> list:
         href_tags = soup.find_all(href=True)
         hrefs = [tag.get("href") for tag in href_tags]
-        scripts = list(map(lambda x: x.get('src'), soup.findAll('script')))
-        if None not in scripts:
-            hrefs.extend(scripts)
+        script_srcs = list(map(lambda x: x.get("src"), soup.findAll("script")))
+        if None not in script_srcs:
+            hrefs.extend(script_srcs)
```

Comment:
Almost-blocking: This looks broken. If a document contains one `<script>` with no src and one `<script>` with a src, then this logic will ignore the `<script>` with a src. Why would we want to exclude the `<script>` with a src?